### DISCUSSION TITLE
API development

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2528] [General]
Hi, community, I would like to know if it 's possible to use this service via API , especially, I input text & image promt and generate images according to the text and image prompts. I wonder if it's possible via API, not manual action. Looking forward to hearing back from you. Thank you.

**Comment by mashb1t** [comment]
Yes, you can find the Websocket API (Gradio) in the footer of the project. Please search discussions and issues for related questions as the same question has been popping up quite a bit.
